### PR TITLE
[#9742] User map updated to 2019 April-May

### LINKED ISSUE #9742: Update user map for 2019 April-May
Similar to #9533
New entries to consider:

Institution | Country
-- | --
Edith Cowan University | Australia
Melbourne Business School | Australia
S P Jain School of Global Management | Australia
The University of Queensland | Australia
AvAnt | Belgium
Hoofdstedelijk Atheneum Karel Buls | Belgium
OLVE College Mortsel | Belgium
PTS Boom | Belgium
Ter Elst | Belgium
Núcleo Regional de Educação de Jacarezinho | Brazil
UTFPR | Brazil
British Columbia Institute of Technology | Canada
British Columbia Institute of Technology | Canada
CDI college | Canada
Cégep Édouard-Montpetit | Canada
Dalhousie | Canada
École Secondaire St-François Xavier | Canada
ES Saint-François-Xavier | Canada
ES Saint-François-Xavier | Canada
ES Saint-François-Xavier | Canada
ES Saint-François-Xavier | Canada
Humber College | Canada
Massey Secondary | Canada
Sheridan College | Canada
Stratford Elementary | Canada
University of Alberta | Canada
University of British Columbia | Canada
Zhejiang University | China
Universidad de los Andes | Colombia
Universidad de los Andes | Colombia
Komercijalna i trgovacka skola Bjelovar | Croatia
Technical University of Liberec | Czech Republic
Universidad San Francisco de Quito | Ecuador
ESCEN - Paris | France
HEC Paris | France
University Koblenz-Landau | Germany
University of Applied Sciences Europe | Germany
Athens University of Economics and Business | Greece
Athens University of Economics and Business | Greece
Dowal School | Honduras
HK Christian School | Hong Kong
Hong Kong Baptist University | Hong Kong
The University of Hong Kong | Hong Kong
Ahemdabad University | India
KIIT University | India
S.P. Jain School of Global Management | India
Institut Teknologi Bandung | Indonesia
Tzu Chi Secondary School | Indonesia
Universitas Pendidikan Ganesha | Indonesia
Universitas Pendidikan Indonesia | Indonesia
Istituto Comprensivo "Gabriele d'Annunzio" | Italy
University of Genoa | Italy
Ritsumeikan Asia Pacific University | Japan
Ritsumeikan Asia Pacific University | Japan
African Launchpad Program | Kenya
Kaunas "Aušros" Gymnasium | Lithuania
Sunway College | Malaysia
Univerisiti Teknologi Malaysia | Malaysia
Universiti Sains Malaysia | Malaysia
Universiti Teknologi Malaysia | Malaysia
Universiti Utara Malaysia | Malaysia
Universidad Autónoma Chapingo | Mexico
Universidad Regiomontana | Mexico
International University of Monaco | Monaco
Tensorflows.org | Morocco
Delft University of Technology | Netherlands
Erasmus Universiteit | Netherlands
Hogeschool Utrecht | Netherlands
ICT Institute | Netherlands
aGreatLife | Philippines
Marist School Marikina | Philippines
Philippine Science High School | Philippines
University of Santo Tomas | Philippines
Universidade do Minho | Portugal
University of Minho | Portugal
Higher School of Economics Saint Petersburg | Russia
University of Caribbean Medical School | San Martin
ITE College West | Singapore
Presbyterian High School | Singapore
Ljubljana University | Slovenia
Cape Peninsula University of Technology | South Africa
EADA Business School | Spain
IES Cap de Lllevant | Spain
Ming Chuan University | Taiwan
Taipei Medical University | Taiwan
GEEN | Turkey
Aston University | UK
Cardiff University | UK
Henley Business School | UK
Sheffield University Management School | UK
Solent University | UK
University of Bradford | UK
University of Glasgow | UK
University of League | UK
University of Reading | UK
University of Worcester | UK
Immaculata University | USA
Alliance Cindy and Bill Simon Tech. Academy High School | USA
Arizona State University | USA
Babson College | USA
Boston College | USA
Boston University | USA
Brigham Young University-Idaho | USA
Brigham Young University, Hawaii | USA
California Baptist University | USA
Central Piedmont Community College | USA
Central York High School | USA
Chapman University | USA
Chapman University | USA
City College of San Francisco | USA
Clemson University | USA
corinnesheltren.com | USA
Cypress Lakes High School | USA
Dalton State College | USA
Del Mar College | USA
Del Norte High School | USA
Delta State University | USA
Drexel University | USA
Duke Global Health Institute | USA
Duke University | USA
Duke University | USA
Engineering and Science University Magnet School | USA
Florida Institute of Technology | USA
Florida International University | USA
Georgia State University | USA
Georgia Tech | USA
Grand Canyon University | USA
Grand Valley State University | USA
Grand Valley State University | USA
GRANICUS, LLC | USA
Hancock Middle-Senior High School | USA
Illinois Institute of Technology | USA
Iowa State University | USA
James Madison University | USA
Kenansville Elementary School | USA
Lake Forest College | USA
Lakeview Centennial High School | USA
Liberty Public Schools | USA
Loyola Marymount University | USA
Luther College | USA
Manatee County Schools | USA
MIT | USA
New Mexico State University | USA
North Dakota State University | USA
North Garland High School | USA
Northeastern University | USA
Northfield Mount Hermon School | USA
Northwest Missouri State University | USA
Northwestern University | USA
Northwood University | USA
Nothern Arizona University | USA
Ocean Township Intermediate School | USA
Ohio University | USA
Penn State Shenango | USA
Phoebus High School | USA
Presbyerian College | USA
Purdue University | USA
Radford University | USA
Rensselaer Polytechnic Institute NROTC | USA
Robert Morris University | USA
Rockford University | USA
Sachse High School | USA
Santa Clara University | USA
Seattle University | USA
Siena College | USA
South Garland High School | USA
Southern College of Optometry | USA
Southern Nazarene University | USA
Springfield College | USA
St. Martin’s University | USA
St. Mary's University of Minnesota | USA
Stranahan High School | USA
Syracuse University | USA
Tallahassee Community College | USA
Tennessee Tech University | USA
The Ohio State University | USA
Union College | USA
Union High School | USA
Union Middle School | USA
United States Military Academy | USA
University at Albany, SUNY | USA
University at Buffalo | USA
University of Alabama at Birmingham | USA
University of Alaska Fairbanks | USA
University of California, Los Angeles | USA
University of Colorado at Boulder | USA
University of Colorado Boulder | USA
University of Dallas | USA
University of Florida | USA
University of Georgia | USA
University of Kansas | USA
University of Massachusetts Dartmouth | USA
University of Minnesota | USA
University of Minnesota - Twin Cities | USA
University of Mississippi | USA
University of Nebraska - Lincoln | USA
University of Nebraska - Lincoln | USA
University of Nevada Las Vegas | USA
University of New England | USA
University of North Georgia | USA
University of North Texas | USA
University of San Francisco | USA
University of South Carolina | USA
University of St. Thomas | USA
University of St. Thomas-Houston | USA
University of Texas at Austin | USA
University of Texas Rio Grande Valley | USA
University of Virginia | USA
UNTHSC | USA
USC Marshall School of Business | USA
Virginia Polytechnic Institute and State University | USA
Virginia Tech | USA
Washington University in St. Louis | USA
Weber State University | USA
Western Washington University | USA
Young Women's Leadership Academy GP | USA

### PR DESCRIPTION
**Pull request by goldami1**
Fixes #9742 

**PR Checklist**

Ensure that you have:
- [x] Read and understood our PR guideline: https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#step-4-submit-a-pr
  - [x] Added the issue number to the "Fixes" keyword above
  - [x] Titled the PR as specified in the abovementioned document
- [x] Made your changes on a branch other than `master` and `release`
- [x] Gone through all the changes in this PR and ensured that:
  - [x] They addressed one (and only one) issue
  - [x] No unintended changes were made
- [x] Run and passed static analysis: `./gradlew lint` and `npm run lint`
- [x] Added/updated tests, if changes in functionality were involved
- [x] Added/updated documentation to public APIs (classes, methods, variables), if applicable

**Outline of Solution**

### CODE REVIEW

File: src/web/data/userMapData.json [hunk 1]
```diff
@@ -1,2538 +1,2630 @@
...
-      "New Mexico State University",
-      "New Prairie United School Corporation",
-      "New Tech Odessa High School",
-      "New York Institute of Technology",
-      "New York University",
-      "Newbury Park High School",
-      "Newport News Schools",
-      "Newton Conover City Schools",
-      "Nichols College",
-      "Noncommsioned Officer
```
Comment (xpdavid) [inline src/web/data/userMapData.json]: The indentation for JSON file is wrong? Should be 2 spaces?

**Comment by xpdavid** [comment]
Close due to inactivity.

File: src/web/data/userMapData.json (hunk 1 above)
Comment (goldami1) [reply src/web/data/userMapData.json]: Correct. beautified to indentation of 1 space as it's in previous commits of this file.

**Comment by goldami1** [comment]
@xpdavid fixed structure of the json file according to your remark. please reopen. thanks

**Comment by xpdavid** [comment]
@goldami1 Have u updated the JSON indentation?

**Comment by goldami1** [comment]
@xpdavid yes, I did in PR #9818

**Pull request closed without merging**